lib/colors.scss

```diff
@@ -2,13 +2,50 @@
 // material design colors https://www.materialui.co/flatuicolors
 // https://material.io/color/#!/?view.left=0&view.right=1&secondary.color=90CAF9&primary.color=ECEFF1
 
+
+//Brand Colors
+$color-basic: (
```

Comment:
We should consider to replace other color-palettes and variables in the near future

Reply:
As discussed we'll do it at a later time. I'll add this as an issue.